week3/yourChoice/src/components/Content/Content.jsx

```diff
@@ -0,0 +1,69 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+
+import { ChooseContainer } from './style/ContentStyles';
+import ContentWrapper from '../Layout/ContentWrapper';
+import ContentHeader from '../Layout/ContentHeader';
+import H1 from '../UI/H1';
+import ChooseBtn from '../UI/ChooseBtn';
+import Button from '../UI/Button';
+import Prefer from './Prefer';
+
+// 전체 content div
+const Content = (props) => {
```

Comment:
props로 넘겨받는거 너무 좋고 여기를 이렇게 바꾸면 아래에서 매번 `props.` 로 접근하지 않아도 되겠죠?! 
```suggestion
const Content = ({isStart, isStartHandler, setGameHandler}) => {
```